docs/.vitepress/config.mts

```diff
@@ -112,7 +116,8 @@ export default withMermaid({
         {
           text: 'Technical Details', link: 'details',
           items: [
-            {text: 'De-Identification', link: '/details/deidentification'}
+            {text: 'De-Identification', link: '/details/deidentification'},
```

Comment:
The two pages De-Identification and Pseudonymization at least partially describe the same stuff, we should merge them.

Reply:
Pseudonymization is part of De-Identification